### PR TITLE
Feature/account unit test

### PR DESCRIPTION
**Pull request by zl730**
# PR Details

## Description

- Add unit test for classes in package `vsys.account`:
  - Address
  - PublicKeyAccount
  - ContractAccount
- Fix some typo

## Related Issue

None

## Motivation and Context

- Increase coverage.
- Coverage of all classes in `vsys.account` is over 85% in this patch
      image

- @Young666666Jian Please refer to this PR

## Types of changes

- [x] Docs change / refactoring / dependency upgrade
- [x] New feature (non-breaking change which adds functionality)

## Checklist

- [x] My code follows the code style of this project.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/test/scala/vsys/account/AddressSpecification.scala [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      val addressBytes = withoutChecksum ++ hash(withoutChecksum).take(Address.ChecksumLength)
+      Address.fromBytes(addressBytes).isRight shouldBe (network == AddressScheme.current.chainId)
+    }
+  }
+
+  property("Address should be invalid for incorrect addressByteLength") {
+    forAll { (data: Array[Byte], random: Byte) =>
+      val withoutChecksum = Address.AddressVersion +: AddressScheme.current.chainId +: random +: hash(data).take(Address.HashLength)
+      val addressBytes = withoutChecksum ++ hash(withoutChecksum).take(Address.ChecksumLength)
+      Address.fromBytes(addressBytes).isRight shouldBe false
```
Comment (ncying) [inline src/test/scala/vsys/account/AddressSpecification.scala]: only for the case `> addressByteLength`, how about random a small non-zero integer number  x (-2, -1, 1, 2) and use `take(Address.HashLength + x)` instead?

Comment (zl730) [reply src/test/scala/vsys/account/AddressSpecification.scala]: Sure

File: src/test/scala/vsys/account/ContractAccountSpecification.scala [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        ContractAccount.tokenIdFromBytes(addressBytes, Ints.toByteArray(0)).isRight shouldBe (random == 0)
+      }
+    }
+  }
+
+  property("Contract Id and Token Id should be consistent") {
+    forAll { (data: Array[Byte]) =>
+      val withoutChecksum = ContractAccount.AddressVersion +: AddressScheme.current.chainId +: hash(data).take(ContractAccount.HashLength)
+      val addressBytes = withoutChecksum ++ hash(withoutChecksum).take(ContractAccount.ChecksumLength)
+      val idxBytes = Ints.toByteArray(0)
```
Comment (ncying) [inline src/test/scala/vsys/account/ContractAccountSpecification.scala]: For this test, idxBytes can be any Integer

Comment (zl730) [reply src/test/scala/vsys/account/ContractAccountSpecification.scala]: ok, my miss

**Pull request merged**